docs/_data/table/en/user-configuration.yml

```diff
@@ -54,9 +54,10 @@ alluxio.user.network.netty.timeout.ms:
 alluxio.user.network.netty.worker.threads:
   How many threads to use for remote block worker client to read from remote block workers.
 alluxio.user.ufs.delegation.enabled:
-  Flag for delegating ufs data operations to the worker. Set this to false to use client to directly
-  communicate with the ufs. Setting this to true is currently experimental and may have an impact
-  on performance for certain workloads.
+  Flag for delegating ufs data operations to the worker, enabled by default. When enabled, the
+  client does not require any under storage libraries. Set this to false to use the client to
```

Comment:
This makes it sound like we don't compile the ufs libraries into the client, but we do, right? Would it make sense to publish a "light" client tarball which only supports delegation?